tests/testthat/test-block.R

```diff
@@ -0,0 +1,17 @@
+test_that("block_append() only writes unique lines", {
+  path <- tempfile()
```

Comment:
Elsewhere in the tests, we use `withr::local_tempfile()` for this purpose, which kills 2 or 3 birds with 1 stone.